timemachine/cpp/src/nonbonded.cu

```diff
@@ -340,15 +352,18 @@ void Nonbonded<RealType, Interpolated>::execute_device(
 
     // do parameter interpolation here
     if(Interpolated) {
-        k_permute_interpolated<<<dimGrid, tpb, 0, stream>>>(
+        CUresult result = compute_permute_interpolated_.configure(dimGrid, 32, 0, stream)
+        .launch(
             lambda,
             N,
             d_perm_,
             d_p,
             d_sorted_p_,
             d_sorted_dp_dl_
         );
-        gpuErrchk(cudaPeekAtLastError());
```

Comment:
This is called on ln 398 after a JIT call, do we still need this?

Reply:
willfix